Source/JavaScriptCore/runtime/TypedArrayType.h

```diff
@@ -141,7 +141,7 @@ inline bool isBigIntTypedView(TypedArrayType type)
     }
 }
 
-inline unsigned logElementSize(TypedArrayType type)
+inline constexpr unsigned logElementSize(TypedArrayType type)
```

Comment:
Hmm, making this `constexpr` seems to have broken WinCairo Debug:

```
JavaScriptCore\runtime\TypedArrayType.h(144): error C3615: constexpr function 'JSC::logElementSize' cannot result in a constant expression
JavaScriptCore\runtime\TypedArrayType.h(166): note: failure was caused by a statement or an expression that is not valid in a constexpr context
```

Reply:
Ah, I guess this has an MSVC-specific solution: https://bugs.webkit.org/show_bug.cgi?id=248026